### DISCUSSION TITLE
Running `now dev` fails with unauthorized

### DISCUSSION THREAD

**Discussion by GMaiolo** [vercel/vercel#3989] [General] [answered]
Hello all,

We just renamed a GitHub repo and deleted the previous one in Zeit with all the integrations to start from scratch, but for some reason we're not able to run `now dev` correctly as it yields the following

```
$ now dev
Now CLI 17.1.1 dev (beta) — https://zeit.co/feedback
Error! Not authorized (403)
```

I'm logged in and am the organization owner, I also switched to the respective team as well and it still yields the same error.

I've tried login in again multiple times.

Any clues on how to solve this?
Thanks in advance

**Comment by mcsdevv** [comment]
Hey @GMaiolo, sorry you were running into this! Are you still experiencing this issue on 18.0.0?

**Reply by GMaiolo** [reply]
Hi @msweeneydev, this has been resolved after creating a support ticket, thanks for reminding me so I can update here!
I needed to delete the `.now` folder, then run `now`, select the correct options to link the project properly and then running `now dev` started to work.

What I needed was pretty much the command `now link` that I don't know if it's released yet.

**Reply by mcsdevv** [reply]
Thanks for letting us know, glad you got it sorted!

**Reply by GMaiolo** [reply]
Thank you, `npm link` would be great for these cases when it exists

**Comment by GMaiolo** [comment, marked as answer]
**Solution:** Needed to delete the `.now` folder, then run `now`, select the correct options to link the project properly and then running `now` dev started to work.

What I needed was pretty much the command `now link` that I don't know if it's released yet.

**Reply by styfle** [reply]
The canary channel has been updated with a new command, `vercel link`.

Try it out now with `npm i -g vercel@canary`, thanks!

**Reply by GMaiolo** [reply]
WIll try it out, thanks @styfle
